x-pack/plugins/secops/public/components/empty_value/index.tsx

```diff
@@ -40,7 +40,7 @@ export const getOrEmptyTag = (path: string, item: unknown): JSX.Element => {
 };
 
 export const getOrEmptyTagFromValue = (value: string | number | null | undefined): JSX.Element => {
-  if (isNil(value)) {
+  if (value == null) {
```

Comment:
so how do we manage undefined if that can be the case

Reply:
The two `==` will detect if it is `null` or if it is `undefined`

For example in the terminal:
```ts
undefined == null
true
```

or in the terminal
```ts
const myFunc = value => value == null;
myFunc(null)
true
myFunc(undefined)
true
myFunc('hi')
false
```

Ref: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness

Reply:
ha ok I understand now, I did not see the `==`, so you just re-write the isNil from lodash https://github.com/lodash/lodash/blob/0843bd46ef805dd03c0c8d804630804f3ba0ca3c/lodash.js#L11974. is that the way we want to go?

Reply:
thanks for the tip

Reply:
The `==` is pretty much everywhere already, and I think most JS devs are used to it so I say yes just rely on `==` instead of `isNil` from lodash.

It's tricky when it comes to when to use `lodash` vs. not to use `lodash`. I can see it both ways with something such as `lodash`'s `isNil`. To a lot of people it has a more semantic name compared to `==`.

However, when it comes to `lodash`, `moment`, or any JavaScript library the way I think about it (personally) is if I can (easily) use a plain vanilla built in JavaScript language feature instead of the library I prefer to use the plain vanilla built as libraries come and go but the language features stick around. For example, in code bases I see people sometimes preferring `ramda` or `fp-ts` instead of `lodash` or `lodash/fp` as their preferred utils toolbox. 

But... if JS adopts a common feature from one of those libs then I usually start changing my style of recommendation and coding style to match that builtin and it makes it easier for my code to adapt to the next toolbox down the road. For example, when it comes to `.reduce`, `.map`, `.set`, `entries` etc... I use JS arrays now. At first they only existed in Lodash, coffeescript, etc... Tons of code bases had different variants of them. Once those functions ended up in the core language of JS it was a lot of sudden changed thinking and changed code reviews where everyone was recommending the array variants. But it turned out for the better in my opinion. If someone was sticking to the lodash variants, most other reviewers would say, 'well, it's fine if you really want to, but I'm going to stick to the built-in ones as they're just there'. Eventually the lodash ones kind of were organically phased out.

Which to me is a good thing since `lodash` can and will remove functions such as `omit` we are seeing.

Now if only JS would adopt `getOr` so I can stop using lodash I would much happier!